source/expressions.tex

```diff
@@ -6011,17 +6015,13 @@
     relational-expression \terminal{>=} compare-expression
 \end{bnf}
 %
-The
-lvalue-to-rvalue\iref{conv.lval},
-array-to-pointer\iref{conv.array},
-and function-to-pointer\iref{conv.func}
-standard conversions are performed on the operands.
+The operands are expected to be prvalues\iref{basic.lval}.
 The comparison is deprecated if
```

Comment:
Ending the previous sentence with a semicolon and continuing with "the comparison is deprecated if [...]" flows better.